module/applications/property-attribution.mjs

```diff
@@ -27,32 +29,45 @@ export default class PropertyAttribution extends Application {
 
   /* -------------------------------------------- */
 
-  /** @inheritDoc */
-  static get defaultOptions() {
-    return foundry.utils.mergeObject(super.defaultOptions, {
-      id: "property-attribution",
-      classes: ["dnd5e", "property-attribution"],
-      template: "systems/dnd5e/templates/apps/property-attribution.hbs",
-      width: 320,
-      height: "auto"
-    });
-  }
+  /** @override */
+  static DEFAULT_OPTIONS = {
+    classes: ["property-attribution"],
+    position: {
+      width: 320
+    }
+  };
 
   /* -------------------------------------------- */
 
+  /** @override */
+  static PARTS = {
+    attribution: {
+      template: "systems/dnd5e/templates/apps/property-attribution.hbs"
+    }
+  };
+
+  /* -------------------------------------------- */
+  /*  Rendering                                   */
+  /* -------------------------------------------- */
+
   /**
    * Prepare tooltip contents.
    * @returns {Promise<string>}
    */
   async renderTooltip() {
-    const data = this.getData(this.options);
-    return (await this._renderInner(data))[0].outerHTML;
+    const options = {};
+    this._configureRenderOptions(options);
+    const context = await this._prepareContext(options);
+    const rendered = await this._renderHTML(context, options);
+    const div = document.createElement("div");
+    div.append(...Object.values(rendered));
+    return div.innerHTML;
```

Comment:
I think we can simplify this by first setting `window.frame: false` in `DEFAULT_OPTIONS`, and adding this `_insertElement` override:

```js
/** @override */
_insertElement(element) {} // do nothing
```

Then this method becomes something like:

```js
async renderTooltip() {
  await this.render();
  return this.element.innerHTML; // Maybe outerHTML, not sure
}
```